docs/src/api.md

```diff
@@ -149,7 +149,7 @@ random_empty
 add_agent_single!
 move_agent_single!
 swap_agents!
-isempty(::Integer, ::ABM)
+isempty(::Int, ::ABM)
```

Comment:
┌ Warning: no docs found for '!!! warning "Missing docstring."
│ 
│     Missing docstring for `isempty(::Int, ::ABM)`. Check Documenter's build log for details.' in `@docs` block in src/api.md:136-153

Reply:
┌ Warning: Cannot resolve @ref for md"[`isempty`](@ref)" in src/api.md.
│ - No docstring found in doc for binding `Base.isempty`.

Reply:
What is the method declaration in the source code? The docstring must match the source code declaration.

Reply:
you need to copy paste this then `Base.isempty(pos::Int, model::ABM{<:DiscreteSpace})`.

Reply:
Done!

Reply:
But fail with error:
https://github.com/JuliaDynamics/Agents.jl/actions/runs/13752193803/job/38454551074?pr=1143#step:8:241